package.json

```diff
@@ -4,6 +4,11 @@
   "version": "0.0.1",
   "description": "A simple atom package to run a UNIX filter on current text",
   "keywords": [],
+  "activationCommands": {
+    "atom-workspace": [
+      "unix-filter:run"
```

Comment:
This wasn't addressed. Also, the commands are defined on `atom-text-editor`, not on `atom-workspace`.